src/server/user-controller.js

```diff
@@ -52,8 +54,15 @@ class UserController {
 
     router.use(passport.initialize());
     router.use(passport.session());
+    router.use(passport.authenticate('apikey', { session: false }));
 
-    passport.use(new LocalStrategy((username, password, cb) => {
+    passport.use('apikey', new ApiKeyStrategy((apikey, cb) => {
+      return apikey === this.serverConfig.importApiKey
```

Comment:
To me it seems like this is authorizing an export since we have the permission "LIST_EXPORTABLE_CONTENT"

Reply:
That makes more sense, yes. I renamed the config key to `exportApiKey`